netmap/types.proto

```diff
@@ -121,16 +129,51 @@ message NodeInfo {
   // Ways to connect to a node
   string address = 2;
 
-  // Attributes of the NeoFS node.
+  // Administrator-defined Attributes of the NeoFS Storage Node.
+  //
+  // Node's attributes are mostly used during Storage Policy evaluation to
+  // calculate object's placement and find a set of nodes satisfying policy
+  // requirements. There are some "well-known" node attributes common to all the
+  // Storage Nodes in the network and used implicitly with default values if not
+  // explicitly set:
+  //
+  // * Capacity \
+  //   Total available disk space in Gigabytes.
+  // * Price \
+  //   Price in GAS tokens for storing one GB of data during one Epoch. In node
+  //   attributes it's a string presenting floating point number with comma or
+  //   point delimiter for decimal part. In the Network Map it will be saved as
+  //   64-bit unsigned integer representing number of minimal token fractions.
+  // * Subnet \
+  //   String ID of Node's storage subnet. There can be only one subnet served
+  //   by the Storage Node.
+  // * Locode \
+  //   Node's geographic location in
+  //   [UN/LOCODE](https://www.unece.org/cefact/codesfortrade/codes_index.html)
+  //   format approximated to the nearest point defined in standard.
+  // * Country \
+  //   Country code in
+  //   [ISO 3166-1_alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2)
+  //   format. Calculated automatically from `Locode` attribute
+  // * Region \
+  //   Country's administative subdivision where node is located. Calculated
+  //   automatically from `Locode` attribute based on `SubDiv` field. Presented
+  //   in [ISO 3166-2](https://en.wikipedia.org/wiki/ISO_3166-2) format.
+  // * City \
+  //   City, town, village or rural area name where node is located written
+  //   without diacritics . Calculated automatically from `Locode` attribute.
+  //
+  // For detailed description of each well-known attribute please see the
+  // corresponding section in NeoFS Technical specification.
   message Attribute {
     // Key of the node attribute.
     string key = 1;
 
     // Value of the node attribute.
     string value = 2;
 
-    // Parent keys, if any
-    // Example: For City it can be Region or Country
+    // Parent keys, if any. For example for `City` it could be `Region` and
+    // `Country`.
```

Comment:
I think only `Country` can be parent for the `City` bucket. It stores direct predecessors, not the whole branch.

Reply:
St.Petersburg looks at you in disbelief.

Reply:
I mean in default case of `Region -> Country -> City` all buckets except first one will have only one parent.

Reply:
Maybe we should use another example here. Geo locations are covered with Locode now. Do you have any proposal? Like `DC->Room->Rack` or availability zones?

Reply:
> Geo locations are covered with Locode now.

Yes, they are. However inner ring nodes will set country, city and region attributes anyways. Are we building network map bucket graph like this one?
```
     Region
      |  \
      |   Country
      |   /
      City   
```

`DC->Room->Rack` should work too as an example.

Reply:
This should be valid. Let's try to look at it in a different issue.